_component_design/left-nav.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Left Navigation
+parent: Navigation
+layout: component
+category: Components
+usage: >
+  Left navigation establishes a user's location within an application and shows the path forward to accomplish a goal. Consistent navigation patterns increase user confidence and efficiency.
+preview-image: preview-images/side-nav.svg
+status: stable
+resource: true
+helix-ui-css: true
+helix-ui-javascript: true
+last-modified: 2017-12-11
+---
+
+<section class="static-section" markdown="1">
+
+## Primary navigation
+
+Use primary navigation for the main sections of an application, or for drop-down categories that organize sections into higher-level groups.
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Left navigation
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Use the left navigation component for web applications, unless there is a firm implementation constraint that requires top navigation.
+
+- This component can extend to fit any navigation information architecture (IA) by scrolling when the list of navigation items exceeds the screen view height. If there are many options to display, the left navigation is scroll-able so that users have access to all items.
+- The preferred implementation includes only primary and secondary navigation links.
+- An alternative, non-preferred implementation includes [tertiary](#tertiary-left-navigation) navigation links.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.cdn_url}}/img/layout/navigation-left.svg)
```

Comment:
Going forward, remove all the empty space surrounding images in the image file.

Reply:
I will remake the images for left-nav in an additional commit today.